### ISSUE TITLE
Can't download >:O

### ISSUE THREAD

**Issue by DaiZaregistriruyus** [space-nuko/ComfyBox#155]
when I'm trying to  download - it says:

Error 404 – Not Found
GitHub produced an error for the download of artifact #885356806.

Usually this means that the artifact has expired (>90 days).

Check on GitHub: https://github.com/space-nuko/ComfyBox/suites/15504393958/artifacts/885356806

**Comment by sinand99** [comment]
Yes, download link is dead

**Comment by halr9000** [comment]
Should be able to install it from source but I am not great with nodejs so I'm not the best to explain how.

Anyone want to try and figure that out?

**Comment by sinand99** [comment]
I think he abandoned this project. He didn't update for 7 months. Comfybox is dead

**Comment by MoJIeAIGC** [comment]
I have found a suitable alternative
https://github.com/xingren23/ComfyFlowApp

**Comment by DaiZaregistriruyus** [comment]
> I think he abandoned this project. He didn't update for 7 months. Comfybox is dead

Why everything I like dies, sinand99? *emoji:sad* *emoji:crying*

**Comment by DaiZaregistriruyus** [comment]
> Should be able to install it from source but I am not great with nodejs so I'm not the best to explain how.
> 
> Anyone want to try and figure that out?

I mean I've tried to do SOMETHING, but I've just wasted my time and space on a hard drive.

**Comment by halr9000** [comment]
I found [the build script](https://github.com/space-nuko/ComfyBox/blob/master/.github/workflows/build-and-publish.yml), it does not look too hard to follow. Probably take 5 min to do it manually once you have node and pnpm installed. Or, someone fork the repo and click the build button. I do not especially care and I'm super tired, or I would. GL.

**Comment by halr9000** [comment]
here i'll get you started

```
winget install OpenJS.NodeJS
winget install pnpm.pnpm
```

YW

**Comment by halr9000** [comment]
> I think he abandoned this project. He didn't update for 7 months. Comfybox is dead

![image](https://github.com/space-nuko/ComfyBox/assets/18482/286178de-c5e6-4b9f-9736-402ccde1099d)

technically August is 3 months ago but yeah hey @space-nuko you still around? we miss you! also

**Comment by sinand99** [comment]
> here i'll get you started
> 
> ```
> winget install OpenJS.NodeJS
> winget install pnpm.pnpm
> ```
> 
> YW

I built it but it doesn't work. I guess it is not compatible with standalone version of ComfyUI. Another problem is that pnpm has downloaded nearly 1GB of files into node_modules folder, which is absurd and unacceptable for a simple frontend app. Not worth the trouble. I'm deleting this crap.

**Comment by ChessScholar** [comment]
I hope this helps someone:
~~https://github.com/ChessScholar/ComfyBox_ReOpened.git~~
Updated as a fork https://github.com/ChessScholar/ComfyBox-ReOpened-Forked-

**Comment by ghostsquad** [comment]
@ChessScholar - Need to enable "issues" on your fork, otherwise, it's hard/impossible for people to communicate. :)

**Comment by halr9000** [comment]
Might wanna do both issues and discussions.  This whole chat in this issue is a great example of what you don't want in an issue. :D Issues should be devoted to the topic of a bug or feature request, and be for/by developers on a project. Discussions are more casual and are better for discussions about a project.

**Comment by soursilver** [comment]
> I hope this helps someone: ~https://github.com/ChessScholar/ComfyBox_ReOpened.git~ Updated as a fork https://github.com/ChessScholar/ComfyBox-ReOpened-Forked-

It helped me... well sort of. When I run localhost:8000 it's just a blank page, looks like something's not loading

**Comment by ChessScholar** [comment]
You guys are awesome, thanks for the patience. Holidays plus getting classwork prepared for my students bogged me down.

I opened up the issues and discussions on the Fork.

**Comment by ChessScholar** [comment]
> > I hope this helps someone: ~https://github.com/ChessScholar/ComfyBox_ReOpened.git~ Updated as a fork https://github.com/ChessScholar/ComfyBox-ReOpened-Forked-
> 
> It helped me... well sort of. When I run localhost:8000 it's just a blank page, looks like something's not loading

I had a similar issue a few times until I got it to work. I think it had something to do with the argument settings in the original comfyui .bat

If I figure it out, I'll let you know.

**Comment by ChessScholar** [comment]
> > I hope this helps someone: ~https://github.com/ChessScholar/ComfyBox_ReOpened.git~ Updated as a fork https://github.com/ChessScholar/ComfyBox-ReOpened-Forked-
> 
> It helped me... well sort of. When I run localhost:8000 it's just a blank page, looks like something's not loading

I figured it out! There was one folder that was recommended I remove because "it looks fishy" #156. Please check the "assets" folder and you'll see all of the necessary js, css, js.maps that will create the page display.

The reason it still worked on my computer was because the folder was still loaded in the browser cache. It wasn't until I removed the folder, deleted my cache and restarted my computer was I able to replicate your blank page problem.

**Comment by johndpope** [comment]
Just launch comfyui with this flag
`  ComfyUI git:(master) ✗ python main.py --enable-cors-header
`

and run the local website
```shell
npm install yarn -g
yarn (will install dependencies)
yarn build
yarn dev
```
